sheepdog/globals.py

```diff
@@ -77,6 +77,45 @@ def case_cache_enabled():
     except (AttributeError, KeyError, TypeError):
         return True
 
+def dictionary_version():
+    """
+    Return the dictionary version. NOTE that the dictionary must be initialized
+    first!
+
+    .. note::
+
+        This function assumes that the dictionary has already been initialized.
+        The except/return None behavior is to, for example, allow Sphinx to
+        still import/run individual modules without raising errors.
+    """
+    from sheepdog import dictionary
+    try:
+        return (
+            'Unknown' if dictionary.settings == None
+            else dictionary.settings.get('_dict_version', 'Unknown')
+        )
+    except (AttributeError, KeyError, TypeError):
```

Comment:
Why return True with an exception?

I think you can remove the try except and just use:
```
# attempt to get dictionary settings, default to empty dict if None or non-existent
settings = getattr(dictionary, 'settings', {}) or {}
return settings.get('_dict_commit', 'Unknown')
```

If you want to always return 'Unknown' even if settings aren't there

Reply:
Copy paste error... I just copied the structure of how it is done in an existing function here